// restaurant.proto
syntax = "proto3";

service RestaurantService {
  rpc GetRestaurants(Empty) returns (RestaurantList);
}

message Empty {}

message Restaurant {
  string name = 1;
  string address = 2;
  float rating = 3;
}

message RestaurantList {
  repeated Restaurant restaurants = 1;
}
